test-butler-app/src/main/java/com/linkedin/android/testbutler/SystemLocaleChanger.java

```diff
@@ -0,0 +1,24 @@
+package com.linkedin.android.testbutler;
+
+import java.util.Locale;
+
+class SystemLocaleChanger {
+    boolean setSystemLocale(String language, String country) {
+        return setSystemLocale(new Locale(language, country));
+    }
+
+    boolean setSystemLocale(Locale locale) {
```

Comment:
Did you read LinkedIn's comment about setting device locale here? [test-butler-library TestButler.java#L196](https://github.com/linkedin/test-butler/blob/master/test-butler-library/src/main/java/com/linkedin/android/testbutler/TestButler.java#L196)

I wonder if that's an issue when trying to set locale very fast for Espresso tests and we haven't encountered it in automation because automation is a fairly slow execution context?